validator/project_validator.go

```diff
@@ -1951,6 +1952,58 @@ func validateGenerateTasks(p *model.Project) ValidationErrors {
 	return validateTimesCalledPerTask(p, ts, evergreen.GenerateTasksCommandName, 1, Error)
 }
 
+func validateModuleUsageInGitGetProject(p *model.Project) ValidationErrors {
+	var errs ValidationErrors
```

Comment:
Lets rename this to ValidationWarnings

Reply:
I don't think we have a validationwarnings struct- other functions which only do warnings have the same nomenclature/struct. What should I do instead?

Reply:
I think since other functions do as such and there's some reusable logic we use for ValidationErrors to store warnings and errors, I think I'll leave this as is unless you think otherwise